src/platform/landing-pages/header-footer-data.json

```diff
@@ -64,58 +64,58 @@
       "href": "https://staging.va.gov/find-forms/",
```

Comment:
Weird, I wonder how come this is `staging.va.gov/find-forms` 🤔

Reply:
Why does that say is the subdomain `staging`?

Reply:
Regardless, I'll update it I think...

Reply:
Looks like it is that way on master....
I'll just fix it. No one ever knew.